tests/Type/QueryPlanTest.php

```diff
@@ -298,6 +298,78 @@ public function testQueryPlan() : void
         self::assertFalse($queryPlan->hasType('Test'));
     }
 
+    public function testQueryPlanForWrappedTypes() : void
+    {
+        $article = new ObjectType([
+            'name'   => 'Article',
+            'fields' => [
+                'id' => ['type' => Type::string()],
+            ],
+        ]);
+
+        $doc               = '
+      query Test {
+        articles {
+            id
+        }
+      }
+';
+        $expectedQueryPlan = [
+            'id' => [
+                'type' => Type::string(),
+                'args' => [],
+                'fields' => [],
+            ],
+        ];
+
+        $hasCalled = false;
+        /** @var QueryPlan $queryPlan */
+        $queryPlan = null;
+
+        $blogQuery = new ObjectType([
+            'name'   => 'Query',
+            'fields' => [
+                'articles' => [
+                    'type'    => Type::nonNull(Type::listOf($article)),
+                    'resolve' => static function (
+                        $value,
+                        $args,
+                        $context,
+                        ResolveInfo $info
+                    ) use (
+                        &$hasCalled,
+                        &$queryPlan
+                    ) : array {
+                        $hasCalled = true;
+                        $queryPlan = $info->lookAhead();
+
+                        return [
+                            ['id' => 5],
+                            ['id' => 10],
+                        ];
+                    },
+                ],
+            ],
+        ]);
+
+        $schema = new Schema(['query' => $blogQuery]);
+        $result = GraphQL::executeQuery($schema, $doc)->toArray();
+
+        self::assertTrue($hasCalled);
+        self::assertEquals(
```

Comment:
That is also superfluous. How about you just return `null` from the resolver and don't even check the result? That makes the test more focused.